e2e.py

```diff
@@ -35,89 +35,130 @@ def load_models_file():
+        )
+    else:
+        connector_object = connector_class_mapping[connector_name][0](
+            api_key=os.environ[connector_class_mapping[connector_name][1]]
+        )
+
+    text2text_models = [model for model in connector["models"] if model["model_type"] == "text2text"]
+    text2vector_models = [model for model in connector["models"] if model["model_type"] == "text2vector"]
+    text2image_models = [model for model in connector["models"] if model["model_type"] == "text2image"]
+
+    if len(text2text_models) > 0:
+        model_object = text2text_models[0]
+        parameters = {}
+        parameters["model"] = model_object["slug"]
+        messages = [{"role": "user", "content": "Hello, how is it going?"}]
+        messages.append({"role": "system", "content": "Behave like Rick Sanchez."})
+        parameters["messages"] = messages
+
+        print(f"Testing model {model_object['slug']} from {connector['provider']} connector \n\n\n")
+        response = connector_object.chat_completion(stream=False, **parameters)
+        print(f"\n\n\n Model {model_object['slug']} succeeed 🚀 \n\n\n")
+
+        response = connector_object.chat_completion(stream=True, **parameters)
+        for text in response:
+            print(connector_object.parse_chunk(text))
+        print(f"\n\n\n Model {model_object['slug']} succeeed with streaming 🚀 \n\n\n")
+
+    if len(text2image_models) > 0:
+        model_object = text2image_models[0]
+
+        parameters = {}
+        parameters["model"] = model_object["slug"]
+        parameters["prompt"] = "A cute baby sea otter"
+        parameters["size"] = "1024x1024"
+        parameters["n"] = 1
+
+        print(f"Testing model {model_object['slug']} from {connector['provider']} connector \n\n\n")
+        response = connector_object.generate_image(**parameters)
+        print(f"\n\n\n Model {model_object['slug']} succeeed 🚀 \n\n\n")
+
+    if len(text2vector_models) > 0:
+        model_object = text2vector_models[0]
+
+        input = "Hello, how is it going?"
+        print(f"Testing model {model_object['slug']} from {connector['provider']} connector")
+        response = connector_object.embeddings(model=model_object["slug"], input=input)
+        print(f"Embeddings: {len(response['data'][0])}")
+        print(f"Model {model_object['slug']} succeeed 🚀 \n\n\n")
+
+
+def run_all_connectors(connector_name_list: list[str] | None = None) -> None:
+    if connector_name_list is None:
+        connectors_mapping = load_models_file()["connectors"]
+        connector_name_list = [connector["provider"] for connector in connectors_mapping]
+
+    for connector in connector_name_list:
+        print("=" * 20, f"Running for Connector: {connector}", "=" * 20)
+        run_single_connector(connector_name=connector)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Testing different providers from prem-utils package")
+
+    parser.add_argument(
```

Comment:
The `all` should happen by default. So, I would include `default="all"`

Reply:
Done